content/2017-munich/sponsor.md

```diff
@@ -69,12 +69,12 @@ prices and benefits:
     </tr>
     <tr>
       <td>
-        Logo on all videos after the event
+        Logo on onsite signage (off-stage)
```

Comment:
The idea (inspired by Katie's comments) was that there would be at least one printed place that has all logos. But yeah, maybe that's getting a bit too dense. Remove it here, but keep it for the normal tiers?